lazysizes-umd.js

```diff
@@ -70,19 +70,36 @@
 		});
 	};
 
-	var triggerEvent = function(elem, name, detail, noBubbles, noCancelable){
-		var event = document.createEvent('Event');
+	var buildEvent = function(name, detail, noBubbles, noCancelable) {
+		// https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill
+		var CustomEvent = window.CustomEvent;
+		if (typeof CustomEvent !== 'function') {
```

Comment:
From what I see `typeof CustomEvent !== 'function'` is not a sufficient way to feature detect. You have to use 

```jstry {
new CustomEvent('f');
} catch(er) {}
```

Reply:
I get it from Mozilla MDN polyfill, but I agree it's not enough and already adopt a `try...catch` strategy as you suggest.